code/modules/mob/living/carbon/human/life/handle_chemicals_in_body.dm

```diff
@@ -32,7 +32,7 @@
 		var/light_amount = 0 //How much light there is in the place, affects receiving nutrition and healing
 		if(isturf(loc)) //Else, there's considered to be no light
 			var/turf/T = loc
-			light_amount = T.get_lumcount(0.5) * 10
```

Comment:
The way the lerp works is that the first argument is the low of the lerp and the second argument the high end of the lerp.

If you do `get_lumcount(0.5, 1)` then a calculated lumcount of `0.75` returns `0.5`.

Then for `get_lumcount(1, 3)`, `2` would result in `0.5`